admin-dev/themes/default/template/controllers/modules/configuration_bar.tpl

```diff
@@ -23,13 +23,24 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *}
 
-{$module_name = $module_name|escape:'html':'UTF-8'}
-{capture}{'/&module_name='|cat:$module_name|cat:'/'}{/capture}
 {if isset($display_multishop_checkbox) && $display_multishop_checkbox}
-<div class="bootstrap panel">
-	<h3><i class="icon-cogs"></i> {l s='Configuration' d='Admin.Global'}</h3>
-	<input type="checkbox" name="activateModule" value="1"{if $module->isEnabledForShopContext()} checked="checked"{/if}
-		onclick="location.href = '{$current_url|regex_replace:$smarty.capture.default:''}&amp;module_name={$module_name}&amp;enable=' + ($(this).prop('checked') ? '1' : '0');" />
-	{l s='Activate module for this shop context: %s.' sprintf=[$shop_context] d='Admin.Modules.Notification'}
+<div class="bootstrap form-horizontal panel">
+	<div class="panel-heading"><i class="icon-cogs"></i> {l s='Configuration' d='Admin.Global'}</div>
+  <div class="form-wrapper">
+    <div class="form-group">
+      <label class="control-label col-lg-4">
+        {l s='Activate module for this shop context: %s.' sprintf=[$shop_context] d='Admin.Modules.Notification'}
+      </label>
+      <div class="col-lg-8">
```

Comment:
It doesn't respect the Bootstrap column system, because you don't use a row as a parent

It's leading to a gutter difference because we don't have the negative margin of the `row`

Reply:
This has to be fixed in helper form then, because I just copied the markup from form. This can be reproduced by configuring any legacy module

<img width="1527" alt="Screenshot 2022-03-23 at 16 20 39" src="https://user-images.githubusercontent.com/1009343/159734516-305fed59-7563-4634-ad2f-7c6cbeeaf8f7.png">
.